config/jobs/kubernetes-sigs/image-builder/image-builder-ci.yaml

```diff
@@ -0,0 +1,30 @@
+postsubmits:
+  kubernetes-sigs/image-builder-konfigadm:
+    - name: post-image-builder-konfigadm
+      labels:
+        preset-dind-enabled: "true"
+        preset-image-builder-gcp-creds: "true"
+        preset-image-builder-aws-creds: "true"
+        preset-image-builder-azure-creds: "true"
+        preset-image-builder-vsphere-creds: "true"
+      branches:
+        - ^master$
+      always_run: false
+      run_if_changed: "^images/konfigadm/"
```

Comment:
this should be more broad for *

Reply:
I think the different testing approaches used for each builder will require a separate job per builder - we also wouldn't want to rebuild all images for all builders on every change